client.go

```diff
@@ -1187,7 +1187,7 @@ func (client *gocloak) GetUsers(token string, realm string, params GetUsersParam
 }
 
 // GetUsersByRoleName returns all users have a given role
-func (client *gocloak) GetUsersByRoleName(token string, realm string, roleName string) ([]*User, error) {
+func (client *gocloak) GetUsersByRoleName(token string, realm string, roleName string, params GetUsersByRoleParams) ([]*User, error) {
```

Comment:
hmm, I've added the params, but I don't see code that uses the params.

What do you think to leave the GetUsersByRoleName without parameter and add a new function GetUsersByRealmRoleName with parameters? So we will keep backward compatibility. Otherwise we will need to create a v5 release.

Reply:
Would you mind if I just removed `GetUsersByRoleParams` from `GetUsersByRoleName` for now but keep it in `GetUsersByClientRoleName`?

Reply:
I'm OK with it